{{cookiecutter.project_slug}}/clients/vue3/src/App.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="font-avenir antialiased text-center">
-    <div class="flex justify-center p-7">
+    <nav class="flex justify-center p-7">
```

Comment:
**Info:** Changed the `<div>` tag to a `<nav>` tag to comply with the landmark requirements as defined by WCAG for compliance.

Screen shot of issue from Axe Dev tools in Chrome attached below:

![Screen Shot 2023-01-04 at 5 31 26 PM](https://user-images.githubusercontent.com/79381287/210662887-45eb7494-0e04-4435-9f79-8b0091c2e924.png)